content/sensu-go/5.18/guides/clustering.md

```diff
@@ -244,11 +244,28 @@ See [Secure Sensu][16] for information about cluster security.
 
 ## Use an external etcd cluster
 
-To use Sensu with an external etcd cluster, you must have etcd 3.3.2 or newer.
-To stand up an external etcd cluster, follow etcd's [clustering guide][2] using the same store configuration.
+You must have etcd 3.3.2 or newer to use Sensu with an external etcd cluster.
+Follow etcd's [clustering guide][2] using the same store configuration to stand up an external etcd cluster.
 
-In this example, you will enable client-to-server and peer communication authentication [using self-signed TLS certificates][13].
-To start etcd for `backend-1` based on the [three-node configuration example][19]:
+To initialize a backend that uses etcd authentication, configure read and write access to the `/sensu.io` key space for your users:
+
+{{< highlight shell >}}
+User: Username
+
+Role sensu_readwrite
+KV Read:
+	[/sensu.io/, <open ended>
+KV Write:
+	[/sensu.io/, <open ended>
+{{< /highlight >}}
+
+Then, create an open-ended grant for the `EtcdRoot`:
+
+{{< highlight shell >}}
```

Comment:
I would put this above the other highlight and highlight that you can use `/opt/etcd/etcdctl user get Username --detail` to get the output to double check the grant is as expected.

Reply:
OK -- let me see if I can get it in the right spot. Thank you.